main.go

```diff
@@ -197,11 +197,7 @@ type _escFile struct {
 }
 
 func (_escLocalFS) Open(name string) (http.File, error) {
-	f, present := _escData[path.Clean(name)]
-	if !present {
-		return nil, os.ErrNotExist
-	}
-	return os.Open(f.local)
```

Comment:
I've thought about it again, and this wouldn't work. The current implementation just strips the prefix if it is present. There is thus ambiguity regarding whether the prefix should be added or not with the above workaround.